init/init.go

```diff
@@ -168,10 +171,21 @@ func getLaunchConfig(cfg *config.CloudConfig, dockerCfg *config.DockerConfig) (*
 	return &launchConfig, args
 }
 
+func isInitrd() bool {
+	var stat syscall.Statfs_t
+	syscall.Statfs("/", &stat)
+	return stat.Type == 0x01021994
```

Comment:
Agree on making it a named constant. This value is the TMPFS_MAGIC
from here: http://man7.org/linux/man-pages/man2/statfs.2.html

I tried to check for RAMFS_MAGIC too, but it's out of range for int32 which is the datatype for stat.Type on arm (it's int64 on amd64). So I had to remove that check for now.